### PR TITLE
Prepare frontend for Sleepers tutorial + script

### LINKED ISSUE #1074: Run Sleepers tutorial nightly
Run Sleepers tutorial nightly in all platforms.

Steps:
- Tester-Puppeteer user needs to have an account because it can't ask for an invitation and it will not be able to confirm its email
- Login
- Use template to create Sleepers study
- Run pipeline
- Check results are there
- Remove Study
- Make sure data was deleted
- Log Out

### LINKED ISSUE #1089: First time a Study is opened, the Nodes Tree is not visible
image

### PR DESCRIPTION
**Pull request by odeimaiz**
## What do these changes do?

This PR updates the client so that it can be controlled by puppeteer. The main goal is to be able to run a tutorial script (sleepers.js) that:
- Register/Login
- Open "Sleepers" template
- Run it
- Check data produced by the first sleeper and make sure it is a number
- Delete the study
- Logout

## Related issue number
child of #1074
closes #1089

## Checklist

- [ ] I think the code is well written
- [ ] Unit tests for the changes exist
- [ ] Documentation reflects the changes
- [ ] **Runs in the swarm**
- [ ] If you design a new module, add your user to .github/CODEOWNERS

### CODE REVIEW

File: ops/travis/e2e-testing/tutorials/sleepers.js [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+  // Use template to create Sleepers study
+  // Run pipeline
+  const templateName = "Sleepers";
+  await auto.dashboardOpenFirstTemplateAndRun(page, templateName);
+  await page.waitFor(5000);
+
+  // Check results are there
+  await auto.openNode(page, 0);
+  // await auto.openLastNode(page);
+  await page.waitFor(5000);
```
Comment (ignapas) [inline ops/travis/e2e-testing/tutorials/sleepers.js]: I am not sure about these waiting times... could lead to false failings. Wouldn't it be better to wait for changes in the interface and continue when specific UI items appear or reach a certain state?

File: ops/travis/e2e-testing/utils/auto.js [hunk 2]
```diff
@@ -278,6 +276,87 @@ async function dashboardDeleteFirstStudy(page) {
...
+  await page.click('[osparc-test-id="openServiceBtn"]');
+}
+
+async function checkDataProducedByNode(page) {
+  console.log("checking Data produced by Node")
+
+  await page.waitForSelector('[osparc-test-id="nodeViewFilesBtn"]')
+  await page.click('[osparc-test-id="nodeViewFilesBtn"]')
+
+  await page.waitFor(10000)
```
Comment (ignapas) [inline ops/travis/e2e-testing/utils/auto.js]: Same here... Why 10s? What if it takes 12 seconds for whatever reason, or maybe it takes 2 seconds and we are wasting time. Is it possible that when the tree loads, it makes the test to continue?

File: services/web/client/source/class/osparc/component/widget/NewStudyDlg.js [hunk 3]
```diff
@@ -87,7 +91,7 @@ qx.Class.define("osparc.component.widget.NewStudyDlg", {
         prjFormLayout.add(templateLayout);
       }
 
-      const createBtn = new qx.ui.form.Button(this.tr("Create"));
+      const createBtn = this.__createBtn = new qx.ui.form.Button(this.tr("Create"));
```
[the remark this answers is not in the document]
Comment (odeimaiz) [reply services/web/client/source/class/osparc/component/widget/NewStudyDlg.js]: Nice

File: .env-devel [hunk 4]
```diff
@@ -35,10 +35,10 @@ RABBIT_PORT=5672
 
 
 REGISTRY_AUTH=True
-REGISTRY_PW=z43
+REGISTRY_PW=adminadmin
 REGISTRY_SSL=True
 REGISTRY_URL=masu.speag.com
-REGISTRY_USER=z43
+REGISTRY_USER=admin
```
Comment (sanderegg) [inline .env-devel]: very good!

Comment (pcrespov) [reply .env-devel]: add user/pass for dev should default to these

File: ops/travis/e2e-testing/demo/puppeteerDemo.js [hunk 5]
```diff
@@ -35,13 +35,15 @@ async function run () {
   // NEW STUDY from Template
```
Comment (pcrespov) [inline ops/travis/e2e-testing/demo/puppeteerDemo.js]: I still think that all the e2e-testing machinery should NOT be under ``osparc-simcore/ops/travis`` but rather under something like ``osparc-simcore/tests/end-to-end``. The former shall be SOLELY for travis scripts and the latter must be runnable in place as well (not only via the travis scripts)

Comment (odeimaiz) [reply ops/travis/e2e-testing/demo/puppeteerDemo.js]: Done!

**Pull request merged**